spark/src/main/java/com/netflix/iceberg/spark/source/Reader.java

```diff
@@ -208,7 +211,10 @@ public Statistics estimateStatistics() {
 
   private List<CombinedScanTask> tasks() {
     if (tasks == null) {
-      TableScan scan = table.newScan().project(lazySchema());
+      TableScan scan = table
+              .newScan()
```

Comment:
Nit: indented 8 instead of 4 from the start of the statement.

Reply:
Fixed.